functors task (1)/student/src/impl.cpp

```diff
@@ -1,2 +1,22 @@
 #include "impl.h"
 
+BeerOrganizer::BeerOrganizer()
+   : currentBeerBand((int)BeerBrand::None)
+{}
+
+BeerBrand BeerOrganizer::operator()()
+{
+    if (++currentBeerBand == (int)BeerBrand::Max) {
+        currentBeerBand = (int)BeerBrand::None;
+        currentBeerBand++;
+    }
+
+    return (BeerBrand)currentBeerBand;
+}
+
+bool isExpensiveBeer(const BeerBrand beerBrand)
+{
+    if (beerBrand == BeerBrand::Corona || beerBrand == BeerBrand::HoeGaarden)
```

Comment:
why not return beerBrand == BeerBrand::Corona || beerBrand == BeerBrand::HoeGaarden?
you should avoid explicit true/false